lib/run.go

```diff
@@ -171,13 +171,14 @@ func findCmdInPath(pathlist []string, cmd, prefix string) (string, error) {
 	}
 
 	for _, p := range pathlist {
-		ex, err := util.Exists(path.Join(prefix, p, cmd))
```

Comment:
I'm actually thinking about just removing the `util.Exists` function. It doesn't really save much code anywhere, since each time I use it I still need to specify what to do if the file doesn't exist, what to do if there was an error, and what to do if the file exists. It doesn't simplify any of the logic.